README.md

```diff
@@ -40,7 +40,7 @@ Create a `workflow.yml` file in your `.github/workflows` directory like:
             uses: "alstr/todo-to-issue-action@v4.0.7"
             id: "todo"
             with:
-              TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              TOKEN: ${{ secrets.GITHUB_TOKEN }} // defaults to ${{ github.token }}
```

Comment:
Maybe we can completely remove this from README.

Reply:
are there valid values other than `secrets.GITHUB_TOKEN`

Reply:
AFAIK, no. Either we can use the secret or `github.token` context to access the `GITHUB_TOKEN` as described [here](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#using-the-github_token-in-a-workflow). Most of the time, 99 times out of 100 people won't need anything other than the default token already available in scope, defaulting to it saves everybody YAML setup mess.